pkg/machine/qemu/machine.go

```diff
@@ -392,7 +392,12 @@ func (v *MachineVM) Stop(name string, _ machine.StopOptions) error {
 		logrus.Warn(err)
 	}
 	// Remove socket
-	return os.Remove(qemuSocketFile)
+	if err := os.Remove(qemuSocketFile); err != nil {
+		return err
+	}
+
+	logrus.Infof("Successfully stopped machine: %s", name)
```

Comment:
Can you use `fmt.Print...`, podman machine start uses `fmt.Printf("Machine %q started successfully\n", vmName)`

Reply:
Done.